plugins/main/server/plugin.ts

```diff
@@ -109,6 +126,113 @@ export class WazuhPlugin implements Plugin<WazuhPluginSetup, WazuhPluginStart> {
     const router = core.http.createRouter();
     setupRoutes(router, plugins.wazuhCore);
 
+    // Register initialization
+    // Index pattern: alerts
+    // TODO: this task should be registered by the related plugin
+    plugins.wazuhCore.initialization.register(
+      initializationTaskCreatorIndexPattern({
+        getIndexPatternID: ctx => ctx.configuration.get('pattern'),
+        taskName: 'index-pattern:alerts',
+        options: {
+          savedObjectOverwrite: {
+            timeFieldName: 'timestamp',
+          },
+          fieldsNoIndices: KnownFields,
+        },
+        configurationSettingKey: 'checks.pattern',
+      }),
+    );
+    // Index pattern: monitoring
+    // TODO: this task should be registered by the related plugin
+    plugins.wazuhCore.initialization.register(
+      initializationTaskCreatorIndexPattern({
+        getIndexPatternID: ctx =>
+          ctx.configuration.get('wazuh.monitoring.pattern'),
+        taskName: 'index-pattern:monitoring',
+        options: {
+          savedObjectOverwrite: {
+            timeFieldName: 'timestamp',
+          },
+          fieldsNoIndices: FieldsMonitoring,
+        },
+        configurationSettingKey: 'checks.monitoring',
+      }),
+    );
+    // Index pattern: vulnerabilities
+    // TODO: this task should be registered by the related plugin
+    plugins.wazuhCore.initialization.register(
+      initializationTaskCreatorIndexPattern({
+        getIndexPatternID: ctx =>
+          ctx.configuration.get('vulnerabilities.pattern'),
+        taskName: 'index-pattern:vulnerabilities-states',
+        options: {
+          fieldsNoIndices: VulnerabilitiesStatesFields,
+        },
+        configurationSettingKey: 'checks.monitoring',
```

Comment:
Should it be vulnerability instead of monitoring?

Reply:
Fixed